js/runner_test.go

```diff
@@ -941,53 +941,6 @@ func TestVUIntegrationTLSConfig(t *testing.T) {
 	}
 }
 
-func TestVUIntegrationHTTP2(t *testing.T) {
-	r1, err := getSimpleRunner("/script.js", `
-			import http from "k6/http";
-			export default function() {
-				let res = http.request("GET", "https://http2.akamai.com/demo");
-				if (res.status != 200) { throw new Error("wrong status: " + res.status) }
-				if (res.proto != "HTTP/2.0") { throw new Error("wrong proto: " + res.proto) }
-			}
-		`)
-	if !assert.NoError(t, err) {
-		return
-	}
-	require.NoError(t, r1.SetOptions(lib.Options{
-		Throw:      null.BoolFrom(true),
-		SystemTags: stats.NewSystemTagSet(stats.TagProto),
-	}))
-
-	r2, err := NewFromArchive(r1.MakeArchive(), lib.RuntimeOptions{})
-	if !assert.NoError(t, err) {
-		return
-	}
-
-	runners := map[string]*Runner{"Source": r1, "Archive": r2}
-	for name, r := range runners {
-		t.Run(name, func(t *testing.T) {
-			samples := make(chan stats.SampleContainer, 100)
-			vu, err := r.NewVU(samples)
-			if !assert.NoError(t, err) {
-				return
-			}
-			err = vu.RunOnce(context.Background())
-			assert.NoError(t, err)
-
-			protoFound := false
-			for _, sampleC := range stats.GetBufferedSamples(samples) {
-				for _, sample := range sampleC.GetSamples() {
-					if proto, ok := sample.Tags.Get("proto"); ok {
-						protoFound = true
-						assert.Equal(t, "HTTP/2.0", proto)
-					}
-				}
-			}
-			assert.True(t, protoFound)
-		})
-	}
-}
-
```

Comment:
Why was this test removed?

Reply:
It is the exactly the same as the other one except it tests if it will work from an archive. I am not against the last part but I would've needed to rewrite it to use the httpmultibin and remove the other. 

And decided against it as I don't think there is a way using it through archive to break something and it is the `js` package that has the most and longest tests (not that the `http` one isn't close behind)

Reply:
OK. It might be better to have the removal as a separate commit and document this reason, but it's not a big deal.